flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceFailover.java

```diff
@@ -137,6 +138,7 @@ protected void assertRecords(Table table, List<Record> expectedRecords, Duration
     SimpleDataUtil.assertTableRecords(table, expectedRecords, timeout);
   }
 
+  @Disabled("Disabled for now as it is flaky on CI")
```

Comment:
do we have an issue to investigate/fix the flakiness? I just want to make sure we don't lose track of enabling this again